consensus/notifications.go

```diff
@@ -71,11 +74,22 @@ func (s *State) notifySubscribers(cc ConsensusChange) {
 	for _, sub := range s.consensusSubscriptions {
 		sub <- cc
 	}
+
+	for _, sub := range s.subscriptions {
+		select {
+		case sub <- struct{}{}:
+			continue
+		default:
+			continue
+		}
```

Comment:
correct. The `continue`s are also unnecessary. I would replace them with comments:
```go
select {
case sub <- struct{}{}:
    // all good
default:
    // channel is full; skip send.
    // This is not a problem, because the channel is merely used to
    // notify the receiver of updates. If the channel is full, it means the
    // receiver has already been informed that the state has changed.
}